### PR TITLE
fix: [PAR-4027] Remove SP on cart page and do not show in order summary

### PR DESCRIPTION
**Pull request by billhefty**
[PAR-4027](https://helloextend.atlassian.net/browse/PAR-4027)

- Do not show SP in order summary on cart page
- When on cart page, remove SP from order
- Add some try/catch around custom js to avoid impacting customer checkout experience

[PAR-4027]: https://helloextend.atlassian.net/browse/PAR-4027?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: view/frontend/layout/checkout_cart_index.xml [hunk 1]
```diff
@@ -6,28 +6,15 @@
...
-                                    <item name="component" xsi:type="string">Extend_Integration/js/view/checkout/summary/shipping-protection</item>
-                                    <item name="sortOrder" xsi:type="string">24</item>
-                                    <item name="config" xsi:type="array">
-                                        <!-- Show custom shipping protection on order summary-->
-                                        <item name="template" xsi:type="string">Extend_Integration/checkout/summary/shipping-protection</item>
-                                        <item name="title" xsi:type="string" translate="true">Shipping Protection</item>
-                                    </item>
-                                </item>
-                                <!--  End-->
+                <arguments>
```
Comment (scott-taylor-extend) [inline view/frontend/layout/checkout_cart_index.xml:9]: @billhefty why are we moving this?

Comment (billhefty) [reply view/frontend/layout/checkout_cart_index.xml:9]: We shouldn't be showing SP in the cart page

Comment (scott-taylor-extend) [reply view/frontend/layout/checkout_cart_index.xml:9]: What if a customer adds it to their checkout but then goes back to the cart, why wouldn't we want it reflected?

File: view/frontend/web/js/view/checkout/summary/shipping-protection-offer.js [hunk 2]
```diff
@@ -23,53 +23,58 @@ define(
...
+                                  // window.location.reload();
+                                }               
+                            })
+                        }
                     }
-                }
-            )
+                ) 
+            } catch (error) {
+                // Swallow error to avoid impacting customer checkout experience
```
[the remark this answers is not in the document]
Comment (aarondeane-extend) [reply view/frontend/web/js/view/checkout/summary/shipping-protection-offer.js:75]: is this to prevent us from borking the rest of the checkout?

**Review by simsben — APPROVED** [review]

**Review by scott-taylor-extend — APPROVED** [review]

**Pull request merged**